ImageFeed/OAuth2/OAuth2Service.swift

```diff
@@ -0,0 +1,117 @@
+import UIKit
+
+
+final class OAuth2Service {
+    static let shared = OAuth2Service()
+    private let urlSession = URLSession.shared
+    private (set) var authToken: String? {
+        get {
+            return OAuth2TokenStorage().token
+        }
+        set {
+            OAuth2TokenStorage().token = newValue
+        } }
+    func fetchOAuthToken(
```

Comment:
любой код с фигурными скобками лучше отделять пустыми строчками
```suggestion

    private(set) var authToken: String? {
        get {
            return OAuth2TokenStorage().token
        }
        set {
            OAuth2TokenStorage().token = newValue
        }
    }
    
    func fetchOAuthToken(
```